test/tests/postprocessors/side_integral/gold/side_integral_functor_refined_out.csv

```diff
@@ -1,3 +1,3 @@
 time,ext_f1,ext_m1,ext_m2,ext_u,ext_v1,ext_v2,int_s1_f1,int_s1_m1,int_s1_m2,int_s1_u,int_s1_v1,int_s2_f1,int_s2_m1,int_s2_m2,int_s2_v2
 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-1,202.0824437408,21.045455689911,132,5.4997921800047,28,0,9.8350801463159,9.8350801463159,84,3.4992744080908,22.5,19.670160292632,19.670160292632,56,45.15
+1,202.0824437408,21.045455689911,132,5.4997921800047,28,0,9.8350801463159,9.8350801463159,84,3.4992744080908,22.5,9.835080146316,9.835080146316,28,22.575
```

Comment:
Did the changes in the last commit necessitate a re-gold?

Reply:
the volume fix ? fi.faceArea()

Reply:
Makes me wonder what was going on with the quadrature rule. But these results look like "gold" results to you? E.g. are they better than before?

Reply:
well the element isnt refined on the side it s looking at. I ll do the analytical calc

Reply:
so m2 is the easiest one to integrate. it's value is 4 over a surface that is a u of length: 3, 1, 3
4 * (3 + 1 + 3) = 28, which is what we get